apps/website/scripts/copy-typedoc.js

```diff
@@ -0,0 +1,49 @@
+const fs = require("fs-extra");
+const path = require("path");
+
+exports.copyTypeDoc = async function () {
+  const workspaceDir = path.resolve(__dirname, "..", "..", "..");
```

Comment:
this is specific to being in `packages/@functionless/[package]`?

Reply:
Yes